Script/char.dat

```diff
@@ -6771,7 +7566,218 @@ punisher
   FriendlyReplies =
   {
     2,
-    "\"Hey, I'm also from New Attnam!\"",
+    "\"We work very hard here. You would not believe how exhausting is it to torture people.\"",
     "\"Gah, that mask itches! But Mr. Decos says we have to wear it to keep the morale of prisoners up.\"";
   }
+
+  Config LAW_STUDENT;
+  {
+    DefaultArmStrength = 10;
+    DefaultLegStrength = 10;
+    DefaultDexterity = 15;
+    DefaultAgility = 15;
+    DefaultEndurance = 10;
+    DefaultPerception = 24;
+    DefaultIntelligence = 28;
+    DefaultWisdom = 25;
+    DefaultCharisma = 30;
+    Sex = FEMALE;
+    HairColor = rgb16(200, 96, 0);
+    EyeColor = rgb16(4, 93, 28);
+    CapColor = rgb16(4, 93, 28);
+    SkinColor = rgb16(240, 200, 201);
+    ClothColor = rgb16(255, 105, 180);
+    ArmMainColor = rgb16(240, 200, 201);
+    LegMainColor = rgb16(240, 200, 201);
+    LegSpecialColor = rgb16(255, 105, 180);
+    BootColor = rgb16(30, 30, 30);
+    HeadBitmapPos = 128, 416;
+    TorsoBitmapPos = 48, 96;
+    ArmBitmapPos = 80, 240;
+    LegBitmapPos = 16, 192;
+    TotalVolume = 40000;
+    TotalSize = 175;
+    CanRead = true;
+    NameSingular = "law student";
+    DefaultName = "Laki Oppilas";
+    RightWielded = 0;
+    LeftWielded = holybook(LEGIFER);
+    KnownCWeaponSkills = { 2, UNARMED, KICK; }
+    CWeaponSkillHits = { 2, 100, 100; }
+    AttachedGod = MELLIS;
+    IsSadist = false;
+    IsNameable = false;
+    IsUnique = true;
+    CanBeCloned = false;
+    IsPolymorphable = false;
+    FriendlyReplies =
+    {
+      8,
+      "\"I'm here on a student exchange program, to gather resources for my dissertation.\"",
+      "\"I'm writing a dissertation on the legal implications of using torture in public proceedings.\"",
+      "\"The law system of Bazzaria is derived from the old, time-tested methods - trials by ordeal, layman juries and lynching.\"",
+      "\"Legifer? No, I'm a future lawyer, of course I worship Mellis!\"",
+      "@Dd says very loudly: \"Why yes, I love working with Mr. Decos! It's a pleasure to assist a true professional with his work.\"",
+      "@Dd whispers with a haunted expression: \"Please, help me! I don't want to be here. Mr. Decos is a madman and those punishers...\"",
+      "@Dd whispers on the verge of tears: \"Please, help me! I don't want to know anything anymore about their laws or techniques. I just want to go home.\"",
```

Comment:
It can optionally be written as one word when it refers to an action stopping:

`I don't want to know about this anymore.` / `I don't want to know about this any more.`

When referring to quantities it's written as two words:

`I don't want to know any more about this.`